envi/archs/rx/emu.py

```diff
@@ -0,0 +1,621 @@
+import struct
```

Comment:
So the PR title says noEMU, so is this file just a work in progress?

Reply:
Correct.  It's a placeholder